.github/actions/run-synthetics-stack-e2e/action.yml

```diff
@@ -0,0 +1,148 @@
+        email: ${{ env.GIT_EMAIL }}
+        token: ${{ env.GITHUB_TOKEN }}
+
+    - uses: elastic/apm-pipeline-library/.github/actions/setup-oblt-cli@current
+      with:
+        github-token: ${{ env.GITHUB_TOKEN }}
+
+    - name: Is an Elastician comment?
+      id: is_elastic_member
+      uses: elastic/apm-pipeline-library/.github/actions/is-member-elastic-org@current
+      with:
+        username: ${{ inputs.user }}
+        token: ${{ env.GITHUB_TOKEN }}
+
+    - name: Notify with a reaction
+      continue-on-error: true
+      uses: elastic/apm-pipeline-library/.github/actions/comment-reaction@current
+      with:
+        repository: ${{ inputs.repository }}
+        commentId: ${{ inputs.comment-id }}
+        emoji: ${{ steps.is_elastic_member.outputs.result && '+1' || '-1' }}
+        token: ${{ env.GITHUB_TOKEN }}
+
+    - name: Exit if not an Elastician comment
+      if: steps.is_elastic_member.outputs.result == 'false'
+      run: exit 1;
+      shell: bash
+
+    - uses: elastic/apm-pipeline-library/.github/actions/docker-login@current
+      with:
+        registry: docker.elastic.co
+        secret: secret/observability-team/ci/docker-registry/prod
+        url: ${{ inputs.vault-url }}
+        roleId: ${{ inputs.vault-role-id }}
+        secretId: ${{ inputs.vault-secret-id }}
+
+    - uses: elastic/apm-pipeline-library/.github/actions/kibana-docker-image@current
+      id: kibana-docker-image
+      with:
+        github-repository: ${{ inputs.repository }}
+        git-ref: refs/pull/${{ inputs.pr-number }}/head
+
+    - name: Create cluster
+      id: create-cluster
+      run: >-
+        ${{ github.action_path }}/create-cluster.sh
+        ${{ steps.kibana-docker-image.outputs.kibana-docker-image }}
+        ${{ steps.kibana-docker-image.outputs.kibana-stack-version }}
+      shell: bash
+
+    - uses: hashicorp/vault-action@v2.7.3
+      with:
+        url: ${{ inputs.vault-url }}
+        method: approle
+        roleId: ${{ inputs.vault-role-id }}
+        secretId: ${{ inputs.vault-secret-id }}
+        exportToken: true
+
+    - name: Get credentials
+      id: get-credentials
+      run: >-
+        ${{ github.action_path }}/get-credentials.sh
+        ${{ steps.create-cluster.outputs.cluster-name }}
+      shell: bash
+
+    - uses: elastic/apm-pipeline-library/.github/actions/buildkite@current
+      id: buildkite
+      with:
+        vaultUrl: ${{ inputs.vault-url }}
+        vaultRoleId: ${{ inputs.vault-role-id }}
+        vaultSecretId: ${{ inputs.vault-secret-id }}
+        pipeline: synthetics-stack-e2e-ci
+        triggerMessage: Triggered by PR ${{ inputs.repository }}#${{ inputs.pr-number }}
+        waitFor: true
+        buildEnvVars: |
+          KIBANA_URL_SECRET=${{ steps.get-credentials.outputs.kibana-url }}
+          KIBANA_USERNAME=${{ steps.get-credentials.outputs.kibana-username }}
+          KIBANA_PASSWORD_SECRET=${{ steps.get-credentials.outputs.kibana-password }}
```

Comment:
The masking technique with the `_SECRET` suffix doesn't work. The environment variable value is visible in the "Environment" tab in the buildkite step.